### PR TITLE
[HUST_CSE] add doxygen missing `@brief`

### PR DESCRIPTION
**Pull request by Nephr1te**
## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)
部分函数注释`@brief`缺失。

#### 你的解决方案是什么 (what is your solution)
添加相应的注释。

#### 在什么测试环境下测试通过 (what is the test environment)
修改注释不会导致不通过。
]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

**Comment by Nephr1te** [comment]
Mind take a look? @mysterywolf @shandianchengzi

File: src/kservice.c [hunk 1]
```diff
@@ -84,7 +84,7 @@ static const char* rt_errno_strs[] =
 };
 
 /**
- * This function return a pointer to a string that contains the
+ * @brief This function return a pointer to a string that contains the
```
Comment (shandianchengzi) [inline src/kservice.c:87]: Please use https://github.com/mysterywolf/formatting to adjust the indent

File: src/kservice.c [hunk 2]
```diff
@@ -346,7 +346,7 @@ rt_weak void *rt_memcpy(void *dst, const void *src, rt_ubase_t count)
 RTM_EXPORT(rt_memcpy);
 
 /**
- * This function will move memory content from source address to destination
+ * @brief  This function will move memory content from source address to destination
```
Comment (shandianchengzi) [inline src/kservice.c:349]: Please use https://github.com/mysterywolf/formatting to adjust the indent

File: src/kservice.c [hunk 3]
```diff
@@ -616,7 +616,7 @@ RTM_EXPORT(rt_strlen);
 #endif /* RT_KSERVICE_USING_STDLIB */
 
 /**
- * The  strnlen()  function  returns the number of characters in the
+ * @brief  The  strnlen()  function  returns the number of characters in the
```
Comment (shandianchengzi) [inline src/kservice.c:619]: Please use https://github.com/mysterywolf/formatting to adjust the indent

**Review by shandianchengzi — CHANGES_REQUESTED** [review]

**Comment by shandianchengzi** [comment]
In next pr, please commit all the new changes in one commit, decrease the commit numbers.

**Comment by shandianchengzi** [comment]
formmatting 工具好像没有解决这个缩进的问题

**Comment by Nephr1te** [comment]
I used the tool to adjust the indent, but it appears that the formatting tool only changed the year at the top of the file. Is this acceptable? @shandianchengzi 
Additionally, I was wondering if there are any plans to utilize clang-format to format ALL the source code?

**Comment by Nephr1te** [comment]
There are various styles of indentation in the code comments. My approach is to follow the indentation of previous tags if there are any, or to use the indentation that has already been used in the current file otherwise.

**Comment by shandianchengzi** [comment]
> I used the tool to adjust the indent, but it appears that the formatting tool only changed the year at the top of the file. Is this acceptable? @shandianchengzi Additionally, I was wondering if there are any plans to utilize clang-format to format ALL the source code?


> There are various styles of indentation in the code comments. My approach is to follow the indentation of previous tags if there are any, or to use the indentation that has already been used in the current file otherwise.


Try to format your comment only, don't clang-format all the code. If clang-format is no use, I think now is acceptable.

**Review by shandianchengzi — APPROVED** [review]

**Comment by Nephr1te** [comment]
Mind take a look? @CuttySark1869

File: src/kservice.c (hunk 2 above)
Comment (mysterywolf) [reply src/kservice.c:349]: formatting 只会将tab转为空格，不会将已有空格问题进行重新调整。  这个空格需要手动处理

Comment (mysterywolf) [reply src/kservice.c:349]: @supperthomas 涛哥，这个空格是不是按照规定是空一格就可以？

**Review by supperthomas — APPROVED** [review]

**Comment by mysterywolf** [comment]
@supperthomas 麻烦review一下，如果没问题可以合并掉。

File: src/kservice.c (hunk 3 above)
Comment (mysterywolf) [reply src/kservice.c:619]: brief 后面空一格即可

File: src/kservice.c (hunk 2 above)
Comment (mysterywolf) [reply src/kservice.c:349]: 看了一下 不需要处理空格，这里留出多个空格是为了和下面保持一致

**Review by mysterywolf — APPROVED** [review]

**Comment by mysterywolf** [comment]
感谢提交PR！

File: src/kservice.c (hunk 2 above)
Comment (shandianchengzi) [reply src/kservice.c:349]: > 看了一下 不需要处理空格，这里留出多个空格是为了和下面保持一致

之前考虑的是@brief  下面那行缩进与@brief  对齐
如
@brief  This
             message

**Pull request merged**